tests/download_test_data.py

```diff
@@ -0,0 +1,51 @@
+import argparse
+import os
+import json
+from mwdblib import Malwarecage, MalwarecageConfig  # type: ignore
+
+
+def main() -> None:
+    parser = argparse.ArgumentParser(
+        description="Downloading test data from MWDB"
+    )
+    parser.add_argument("mwdb_user", help="Mwdb username")
+    parser.add_argument("mwdb_pass", help="Mwdb password")
+    parser.add_argument(
+        "mwdb_config_id", nargs="?", help="Config Id", default=""
+    )
+    args = parser.parse_args()
+
+    current_path = os.path.abspath(os.path.dirname(__file__))
+    testdir = current_path + "/testdata/"
+
+    mwdb = Malwarecage()
+    mwdb.login(args.mwdb_user, args.mwdb_pass)
+
+    if args.mwdb_config_id:
+        mwdb_config = mwdb.query_config(args.mwdb_config_id)
+
+        json_file_name = (
+            mwdb_config.family + "_" + args.mwdb_config_id + ".json"
+        )
+        generate_config_json_file(testdir, json_file_name, mwdb_config)
+    else:
+        families_parsed = {""}
```

Comment:
literally `set()` is better IMO (empty set, instead of a "set with en empty string")